ckanext/dcat/controllers.py

```diff
@@ -45,6 +46,26 @@ def read_catalog(self, _format=None):
         except toolkit.ValidationError, e:
             toolkit.abort(409, str(e))
 
+    def read_organization(self, _id, _format=None):
+
+        if not _format:
+            _format = check_access_header()
+
+        if not _format:
+            return OrganizationController().read(_id)
+
+        toolkit.response.headers.update(
+            {'Content-type': CONTENT_TYPES[_format]})
+
+        try:
+            fq = ' organization:"'+_id+'"'
```

Comment:
Don't add the `fq` param here, just pass the `id` to the action (see comment below)